src/main/java/uk/ac/cam/cl/dtg/isaac/dao/EventBookingPersistenceManager.java

```diff
@@ -79,6 +79,17 @@ public EventBookingPersistenceManager(final PostgresSqlDb database, final UserAc
         return convertToDTO(Lists.newArrayList(dao.findAllByUserId(userId)));
     }
 
+    /**
+     * @param userId
+     *            - user of interest.
+     * @return events
+     * @throws SegueDatabaseException
+     *             - if an error occurs.
+     */
+    public List<DetailedEventBookingDTO> getEventReservationsByUserId(final Long userId) throws SegueDatabaseException {
+        return convertToDTO(Lists.newArrayList(dao.findAllReservationsByUserId(userId)));
```

Comment:
Does this augment the booking objects with PII? If so, are you removing them as they bubble up?

Reply:
Yeah, I don't really like it but in the facade it takes this returned thing and uses the event Ids to get the IsaacEventPageDTO in `getEventsReservedByUser` and never returns the actual reservations. I couldn't immediately see a `convertToDTO` that went to `EventBookingDTO` - I'll double check though

Reply:
Worth double checking but if you are sure you are not returning those objects then we're probably safe.

Reply:
Done!